### ISSUE TITLE
missing CRLF when adding name at the end of a subroutine

### ISSUE THREAD

**Issue by davidpfister** [PlasmaFAIR/fortitude#223]
On Windows 10, with fortitude 0.6.1, I got the following when trying to fix some issues
![image](https://github.com/user-attachments/assets/839e55ff-9e93-4ac4-a0dc-f1b1f5c6b765)

The CRLF is missing at the end of the subroutine name and the `end module` gets merged with the name

**Comment by LiamPattinson** [comment]
Thanks for catching this! I can replicate it on Linux too. Our tests were missing it because we'd put 'helpful' comments at the end of our test lines:

```f90
module mymod1
  implicit none
  type mytype
    integer :: x
  end type                      ! catch this
contains
  subroutine mysub1()
    write (*,*) 'hello world'
  end subroutine                ! catch this
  subroutine mysub2()
    write (*,*) 'hello world'
  end subroutine mysub2         ! ignore this
end                             ! catch this
```

This would delete only up to the comment, so we weren't seeing it that it would otherwise delete the next newline too :man_facepalming:.

We'll try to have this fixed as soon as possible!

**Comment by LiamPattinson** [comment]
This should now be fixed in v0.6.2, apologies for any damages!

This is our first release of auto-fixes, so I was expecting to hear of a few bugs, but I'm a bit embarrassed that we missed this one. In future, I might see if we can hide new fixes behind `--preview` so that only users willing to experiment with unstable features are likely to encounter bugs like this.
